src/services/interview.ts

```diff
@@ -6,13 +6,24 @@ import {
   RawInterviewAvailabilityData,
   HydratedInterview,
   RawMemberData,
-} from '../types';
-import api from './api';
-import { deserializeMember } from './member';
+  RawInterViewPoolStatus,
+  InterviewPoolStatus,
+} from "../types";
+import api from "./api";
+import { deserializeMember } from "./member";
 import {
   deserializeTechnicalQuestion,
   deserializeBehavioralQuestion,
-} from './question';
+} from "./question";
+
+function serializeInterviewPoolStatus(
+  data: RawInterViewPoolStatus
```

Comment:
Let's use the same format we've used elsewhere for serializing and deserializing.